extensions/blocks/subscriptions/edit.js

```diff
@@ -1,81 +1,325 @@
+					/>
+
+					<RangeControl
+						value={ borderWeight }
+						label={ __( 'Border Weight', 'jetpack' ) }
+						min={ MIN_BORDER_WEIGHT_VALUE }
+						max={ MAX_BORDER_WEIGHT_VALUE }
+						initialPosition={ INITIAL_BORDER_WEIGHT_POSITION }
+						allowReset
+						onChange={ newBorderWeight => setAttributes( { borderWeight: newBorderWeight } ) }
+					/>
+
+					<PanelColorSettings
+						title={ __( '', 'jetpack' ) }
+						className="wp-block-jetpack-subscriptions__bordercolorpanel"
+						colorSettings={ [
+							{
+								value: borderColor.color,
+								onChange: setBorderColor,
+								label: __( 'Border Color', 'jetpack' ),
+							},
+						] }
+					/>
+				</PanelBody>
+
+				<PanelBody
+					title={ __( 'Spacing Settings', 'jetpack' ) }
+					initialOpen={ false }
+					className="wp-block-jetpack-subscriptions__spacingpanel"
+				>
+					<RangeControl
+						value={ borderRadius }
+						label={ __( 'Space Inside', 'jetpack' ) }
+						min={ MIN_BORDER_RADIUS_VALUE }
+						max={ MAX_BORDER_RADIUS_VALUE }
+						initialPosition={ INITIAL_BORDER_RADIUS_POSITION }
+						allowReset
+						onChange={ newBorderRadius => setAttributes( { borderRadius: newBorderRadius } ) }
+					/>
+
+					<RangeControl
+						value={ borderRadius }
+						label={ __( 'Space Between', 'jetpack' ) }
+						min={ MIN_BORDER_RADIUS_VALUE }
+						max={ MAX_BORDER_RADIUS_VALUE }
+						initialPosition={ INITIAL_BORDER_RADIUS_POSITION }
+						allowReset
+						onChange={ newBorderRadius => setAttributes( { borderRadius: newBorderRadius } ) }
+					/>
+				</PanelBody>
 
-	onChangeSubmit( submitButtonText ) {
-		this.props.setAttributes( { submitButtonText } );
-	}
+				<PanelBody
+					title={ __( 'Display Settings' ) }
+					initialOpen={ false }
+					className="wp-block-jetpack-subscriptions__displaypanel"
+				>
+					<ToggleControl
+						label={ __( 'Show subscriber count', 'jetpack' ) }
+						checked={ showSubscribersTotal }
+						onChange={ () => {
+							setAttributes( { showSubscribersTotal: ! showSubscribersTotal } );
+						} }
+					/>
+
+					<ToggleControl
+						label={ __( 'Place button on new line', 'jetpack' ) }
+						checked={ showSubscribersTotal }
+						onChange={ () => {
+							setAttributes( { showSubscribersTotal: ! showSubscribersTotal } );
+						} }
+					/>
+				</PanelBody>
+			</InspectorControls>
+
+			<div className={ className } role="form">
+				{ showSubscribersTotal && <p role="heading">{ subscriberCountString }</p> }
+
+				<TextControl
+					placeholder={ subscribePlaceholder }
+					disabled={ true }
+					className={ classnames( classes, 'wp-block-jetpack-subscriptions__textfield' ) }
+					style={ styles }
+				/>
+
+				<RichText
+					allowedFormats={ [] }
+					className={ classnames( classes, 'wp-block-jetpack-subscriptions__button' ) }
+					onChange={ value => setAttributes( { submitButtonText: value } ) }
+					placeholder={ __( 'Add text…', 'jetpack' ) }
```

Comment:
We shouldn't need the `placeholder` attribute at all since `value` has a default defined `Subscribe`.

Reply:
Done.